ui/src/components/People/DetailsCard.vue

```diff
@@ -370,10 +372,16 @@ export default {
       } else {
         for (key in fields) {
           let field = fields[key];
-          fields[key].value = this.data[index][field.title];
+          if(fields[key].type == "dateTime" && this.data[index][field.title.split(".")[0]]!=null)
```

Comment:
Can you add some comments here and below explain what's happening? It's not immediately clear based on the code.